libp2p/standard_setup.nim

```diff
@@ -54,7 +48,7 @@ proc newStandardSwitch*(privKey = none(PrivateKey),
     of SecureProtocol.Noise:
       secureManagerInstances &= newNoise(rng, seckey).Secure
     of SecureProtocol.Secio:
-      secureManagerInstances &= newSecio(rng, seckey).Secure
+      quit("Secio is deprecated!") # use of secio is unsafe
```

Comment:
instead, might be able to mark the  `Secio` enum value with `{.deprecated.}` - if not, might want to provide instructions for how to use secio anyway since it's needed to access old go implementations like ipfs etc

Reply:
AFAIK, `secio` has been removed everywhere, since it's been marked insecure

- js https://github.com/ipfs/js-ipfs/commit/5f5ef7ee6cc6dc634cc6adbede0602492490a85d
- go https://github.com/ipfs/go-ipfs/commit/1a0474bb0d3821d75573bc990267841fcb729778